docs/tutorials/substreams/getting-started.experimental.md

```diff
@@ -0,0 +1,390 @@
+  replicaCount: 1
+  statefulset:
+    commonOneBlockStoreUrl: "file:///storage/one-blocks"
+    commonMergedBlocksStoreUrl: "file:///storage/merged-blocks"
+    commonForkedBlocksStoreUrl: "file:///storage/forked-blocks"
+    substreamsStateStoreUrl: "file:///storage/state-store"
+  # enabled local storage to replace gloud storage
+  localVolume:
+    enabled: true
+firehose:
+  replicaCount: 1
+  statefulset:
+    commonOneBlockStoreUrl: "file:///storage/one-blocks"
+    commonMergedBlocksStoreUrl: "file:///storage/merged-blocks"
+    commonForkedBlocksStoreUrl: "file:///storage/forked-blocks"
+  # enabled local storage to replace gloud storage
+  localVolume:
+    enabled: true
+
+```
+
+Some important configurations you need to know:
+
+- `statefulset: persistentVolumeClaimEnabled = false`: This will disable the persistent volume claim which is only used in Ultra's deployment.
+- `localVolume: enabled = true`: This will enable local storage to replace cloud storage, this will make use of `minikube` persist file store. You can read more here [Persistent Volumes](https://minikube.sigs.k8s.io/docs/handbook/persistent_volumes/).
+- `p2p-peer-address = producer-0.producer.default.svc.cluster.local:9876`: you can see why I mentioned the importance of this config above. If you use a different name/namespace please update the according to the format mentioned in [DNS Pod](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pods) or simply use get pod IP and replace it here.
+
+The final step is to install `firehose-antelope` chart
+
+```sh
+helm install firehose-antelope ./firehose-antelope -f ./values.yaml
+```
+If the deployment has no error, you should see `READY` at `1/1` (please ignore `firehose` ready status for now).
+```sh
+kubectl get pod
+```
+```sh
+NAME                       READY   STATUS    RESTARTS   AGE
+firehose-0                 0/1     Running   0          2m27s
+merger-0                   1/1     Running   0          2m27s
+producer-0                 1/1     Running   0          3m19s
+reader-node-0              1/1     Running   0          2m27s
+relayer-7fc86d45cc-dz8vz   1/1     Running   0          2m27s
+substreams-tier1-0         1/1     Running   0          2m27s
+substreams-tier2-0         1/1     Running   0          2m27s
+```
+### Deploy `substreams-charts` to Google Kubernetes Engines (GKE) cluster
+
+**NOTE**: If you are familiar with deployment with `GKE`, once you log in and connect to your project, you can use the same `helm` commands to deploy to `GKE`.
+
+If not familiar with GKE, you can follow these tutorials:
+
+- Installing [Google Cloud SDK](https://cloud.google.com/sdk/docs/install)
+- Create a cluster and [deploy an app to GKE cluster](https://cloud.google.com/kubernetes-engine/docs/deploy-app-cluster)
+
+Once you can deploy an app on your own GKE cluster, you can use the same `helm` commands provided above to deploy `substreams-charts`.
+
+## How to consume `substreams` data
```

Comment:
`substreams` should be `Substreams`

Reply:
updated

Reply:
You updated another line, which is good, but this line has not been updated.